runtime/kusama/src/lib.rs

```diff
@@ -109,6 +110,16 @@ impl Filter<Call> for BaseFilter {
 pub struct IsCallable;
 frame_support::impl_filter_stack!(IsCallable, BaseFilter, Call, is_callable);
 
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
why the feature gating? benchmarks should be reflective of the on-chain logic. and in any case, i would want the on-chain logic to be the `EnsureOneOf` value.